fv3gfs/util/_timing.py

```diff
@@ -66,9 +71,22 @@ def times(self) -> Mapping[str, float]:
             )
         return self._accumulated_time.copy()
 
+    @property
+    def hitcounts(self) -> Mapping[str, int]:
```

Comment:
Please rename this to `hits` to match `times`. It should be clear enough from the type being integer that these are counts.